ra/ra.go

```diff
@@ -390,6 +402,12 @@ func (ra *RegistrationAuthorityImpl) validateContacts(ctx context.Context, conta
 		if parsed.Scheme != "mailto" {
 			return berrors.MalformedError("contact method %s is not supported", parsed.Scheme)
 		}
+		if addressComponents := strings.Split(contact, "@"); len(addressComponents) != 2 {
```

Comment:
This should probably happen in `validateEmail`, then we can use the result of `mail.ParseEmail` to check the domain and not break people with wild 5322 addresses.

Reply:
Makes sense to me :+1: Moved in 808a996